src/SdfEntityCreator.cc

```diff
@@ -379,6 +380,32 @@ Entity SdfEntityCreator::CreateEntities(const sdf::Light *_light)
   this->dataPtr->ecm->CreateComponent(lightEntity,
       components::Name(_light->Name()));
 
+  if (_light->Type() == sdf::LightType::POINT)
+  {
+    this->dataPtr->ecm->CreateComponent(lightEntity,
+        components::LightType(std::string("Point")));
+  }
+  else if (_light->Type() == sdf::LightType::DIRECTIONAL)
+  {
+    this->dataPtr->ecm->CreateComponent(lightEntity,
+        components::LightType(std::string("Directional")));
+  }
+  else if (_light->Type() == sdf::LightType::SPOT)
+  {
+    this->dataPtr->ecm->CreateComponent(lightEntity,
+        components::LightType(std::string("Spot")));
```

Comment:
How about creating a `convert` function from `sdf::LightType` to `std::string` in `Conversions.hh`? It would be better to reuse the same logic in many places. I'm concerned that different plugins may use different strings (i.e. `spot`, `SPOT`...).

I can even see this function living in libSDFormat.

Also, I think that we should store the types as lowercase. The GUI can capitalize them as needed.